test/test_utils.py

```diff
@@ -412,6 +413,7 @@ def test_cpu(self):
 
     @unittest.skipIf(not HAS_CFFI or not HAS_CUDA, "ffi tests require cffi package")
     @unittest.skipIf(IS_WINDOWS, "ffi doesn't currently work on Windows")
+    @unittest.skipIf(BUILT_WITH_ROCM and not OVERRIDE_DISABLED_UNIT_TESTS_ROCM, "Test disabled for ROCm")
```

Comment:
I would change the message to "test doesn't currently work on the ROCm stack" to be in line w/ the windows message

Reply:
Can you make a helper function for this `skipIf` decorator; I'd prefer if we didn't copy paste `BUILT_WITH_ROCM and not OVERRIDE_DISABLED_UNIT_TESTS_ROCM` everywhere

Reply:
Given the discussion in other comments on this PR about removing the OVERRIDE define, would you still want a helper function? I could see a little value in it in case the condition changes, but that's it.

Reply:
@jithunnair-amd No need for the helper.